core/src/main/java/org/stellar/anchor/auth/AuthHelper.java

```diff
@@ -59,4 +60,20 @@ public AuthHeader<String, String> createPlatformServerAuthHeader() throws Invali
         return null;
     }
   }
+
+  @Nullable
+  public AuthHeader<String, String> createCallbackAuthHeader() throws InvalidConfigException {
```

Comment:
This function is highly similar to [`createPlatformServerAuthHelper()`](https://github.com/stellar/java-stellar-anchor-sdk/pull/909/files#diff-9196d7e109d758f964b6d72470e61b9deaf835a0b71b21120c4a797366b41520R50-R62). 

Can we benefit from extracting to another function?

Reply:
Done